Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -29,11 +37,52 @@ public Task<bool> Execute(EditUserRequest request)
                     || string.IsNullOrEmpty(passwordRequest.NewPassword)))
             {
                 passwordChangeValidator.ValidateAndThrow(passwordRequest);
+            } else
+            {
+                throw new BadRequestException("Password is null or empty");
```

Comment:
We can edit password or not . And it wouldn't be bad if we stay one model null

Reply:
Fixed